parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -507,7 +534,9 @@ configGPUDrivers() {
   retrycmd_if_failure 120 5 25 pkill -SIGHUP dockerd || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_CONFIG"}}
   mkdir -p $GPU_DEST/lib64 $GPU_DEST/overlay-workdir
   retrycmd_if_failure 120 5 25 mount -t overlay -o lowerdir=/usr/lib/x86_64-linux-gnu,upperdir=${GPU_DEST}/lib64,workdir=${GPU_DEST}/overlay-workdir none /usr/lib/x86_64-linux-gnu || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_CONFIG"}}
-  retrycmd_if_failure 3 1 600 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_START_FAIL"}}
+  export -f installGPUDriversRun
+  export GPU_DEST GPU_DV
```

Comment:
do we need to export this?

Reply:
yes, this is run as a bash sub-command by the retry entrance,
we need to export it.

Reply:
oh, I mis-read it, these are two vars not a single one :)

can we pass them in as parameters to the method? global var will some time cause trouble.